bin/container_daemon

```diff
@@ -30,6 +30,12 @@ else
     *) INIT_ARGS="--profile=$IPFS_PROFILE" ;;
   esac
   btfs init $INIT_ARGS
+  if [ "$BTFS_ENV" != "" ]; then
+    btfs config Services.EscrowDomain escrow-service:50051
+    btfs config Services.GuardDomain guard-interceptor-service:50051
+    btfs config Services.HubDomain query-service:50051
+    btfs config Services.StatusServerDomain status-grpc-service:50051
```

Comment:
Not understand these 4 lines, seems it only can work in our k8s, is that expected result?

Reply:
it is only set in repairer node with the BTFS_ENV set